core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@crate.ai/discogs-sdk",
-  "version": "2.1.5",
+  "version": "2.2.3",
```

Comment:
Not that I care much but how did you go from 2.1.5 to 2.2.3

Reply:
had some git shenanigans